### ISSUE TITLE
Is it possible to use it from python?

### ISSUE THREAD

**Issue by ariaaan** [AlexeyAB/darknet#163]
I have a detection pipeline implemented in python, I'm using darknet and a keras classification model. But when using the official darknet with a trained tiny yolo I'm getting ~25fps, and with this fork I get ~100fps, (I'm using a GTX 1070). 

Is there any way to use this fork from python?

Thanks!

**Comment by arevel190** [comment]
I am also curious if it is possible to use t from Python.

**Comment by malehovich** [comment]
Хочу поблагодарить за отличную оптимизацию по скорости. И также присоединиться к вопросу о коннекторе для питона. ))

**Comment by AlexeyAB** [comment]
@ariaaan @arevel190 @malehovich Yes, you can use it from the Python: https://github.com/AlexeyAB/darknet/issues/621#issuecomment-380922711
